util/configure

```diff
@@ -638,7 +638,7 @@ _END_
             my $ofile = tmpnam();
             my $comp = ($cc || 'cc');
 
-            if (system("$comp -o $ofile -msse4.2 -c $cfile") == 0 && -s $ofile) {
+            if (system("grep -q sse4_2 /proc/cpuinfo") && system("$comp -o $ofile -msse4.2 -c $cfile") == 0 && -s $ofile) {
```

Comment:
Hmm, `/proc/cpuinfo` is not portable. We need to support various different operating systems like Windows, FreeBSD, and Mac OS X. Maybe we should actually run the resulting C program to validate?